tensorflow-examples/src/main/java/org/tensorflow/model/examples/objectdetection/FasterRcnnInception.java

```diff
@@ -0,0 +1,302 @@
+detection_anchor_indices: a tf.float32 tensor of shape [N] and contains the anchor indices of the detections after NMS.
+detection_multiclass_scores: a tf.float32 tensor of shape [1, N, 90] and contains class score distribution (including background) for detection boxes in the image including background class.
+
+However using
+venv\Scripts\python.exe venv\Lib\site-packages\tensorflow\python\tools\saved_model_cli.py show --dir models\faster_rcnn_inception_resnet_v2_1024x1024 --all
+2021-03-19 12:25:37.000143: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
+
+MetaGraphDef with tag-set: 'serve' contains the following SignatureDefs:
+
+signature_def['__saved_model_init_op']:
+  The given SavedModel SignatureDef contains the following input(s):
+  The given SavedModel SignatureDef contains the following output(s):
+    outputs['__saved_model_init_op'] tensor_info:
+        dtype: DT_INVALID
+        shape: unknown_rank
+        name: NoOp
+  Method name is:
+
+signature_def['serving_default']:
+  The given SavedModel SignatureDef contains the following input(s):
+    inputs['input_tensor'] tensor_info:
+        dtype: DT_UINT8
+        shape: (1, -1, -1, 3)
+        name: serving_default_input_tensor:0
+  The given SavedModel SignatureDef contains the following output(s):
+    outputs['detection_anchor_indices'] tensor_info:
+        dtype: DT_FLOAT
+        shape: (1, 300)
+        name: StatefulPartitionedCall:0
+    outputs['detection_boxes'] tensor_info:
+        dtype: DT_FLOAT
+        shape: (1, 300, 4)
+        name: StatefulPartitionedCall:1
+    outputs['detection_classes'] tensor_info:
+        dtype: DT_FLOAT
+        shape: (1, 300)
+        name: StatefulPartitionedCall:2
+    outputs['detection_multiclass_scores'] tensor_info:
+        dtype: DT_FLOAT
+        shape: (1, 300, 91)
+        name: StatefulPartitionedCall:3
+    outputs['detection_scores'] tensor_info:
+        dtype: DT_FLOAT
+        shape: (1, 300)
+        name: StatefulPartitionedCall:4
+    outputs['num_detections'] tensor_info:
+        dtype: DT_FLOAT
+        shape: (1)
+        name: StatefulPartitionedCall:5
+    outputs['raw_detection_boxes'] tensor_info:
+        dtype: DT_FLOAT
+        shape: (1, 300, 4)
+        name: StatefulPartitionedCall:6
+    outputs['raw_detection_scores'] tensor_info:
+        dtype: DT_FLOAT
+        shape: (1, 300, 91)
+        name: StatefulPartitionedCall:7
+  Method name is: tensorflow/serving/predict
+
+Defined Functions:
+  Function Name: '__call__'
+    Option #1
+      Callable with:
+        Argument #1
+          input_tensor: TensorSpec(shape=(1, None, None, 3), dtype=tf.uint8, name='input_tensor')
+
+So it appears there's a discrepancy between the web page and running saved_model_cli.
+*/
+
+import org.tensorflow.*;
```

Comment:
We prefer explicit imports over star imports.

Reply:
Will do.